app/Services/VoucherEvaluator/EvaluatorFactory.php

```diff
@@ -51,106 +38,69 @@ public static function make($type = null, Carbon $offsetDate = null)
-                ]
-            ],
-            Family::class => [
-                'credits' => [
-                    new FamilyIsPregnant(null, 3)
-                ],
-                'notices' => [],
-                'relations' => ['children'],
-            ],
-            Registration::class => [
-                'credits' => [],
-                'notices' => [],
-                'relations' => ['family'],
-            ],
-        ];
+        // Look up Sponsor rules specific to our Registration
+        $evaluations = $registration->centre->sponsor->evaluations;
+        return self::make($evaluations, $offsetDate);
     }
 
     /**
-     * Creates the correct rule for an "extended age" evaluation
+     * Combines the standard evaluations with specific modifications.
      *
+     * @param Collection $modEvaluations
      * @param Carbon $offsetDate
      * @return array
      */
-    private static function createExtendedEvaluations(Carbon $offsetDate)
+    public static function generateEvaluations(Collection $modEvaluations, Carbon $offsetDate)
     {
-        return $evaluations = [
-            Child::class => [
+        $evaluations = [
+            "App\Child" => [
                 'credits' => [
-                    new ChildIsUnderOne($offsetDate, 3),
-                    new ChildIsUnderSecondarySchoolAge($offsetDate, 3)
+                    "ChildIsUnderOne" => new ChildIsUnderOne($offsetDate, 6),
+                    "ChildIsBetweenOneAndPrimarySchoolAge" => new ChildIsBetweenOneAndPrimarySchoolAge($offsetDate, 3),
                 ],
                 'notices' => [
-                    // warn if we're almost 1yo
-                    new ChildIsAlmostOne($offsetDate),
-
-                    // ... or if under school age; turns out we don't need this here!
-                    # new ChildIsUnderSchoolAge($offsetDate),
-
-                    // ... or if almost primary school age
-                    new ChildIsAlmostSchoolAge($offsetDate),
-
-                    // ... or if actually primary school age; turns out we don't need this here!
-                    # new ChildIsSchoolAge($offsetDate)
-
-                    // ... or if almost secondary school age
-                    new ChildIsAlmostSecondarySchoolAge($offsetDate),
+                    "ChildIsAlmostOne" => new ChildIsAlmostOne($offsetDate, 0),
+                    "ChildIsAlmostPrimarySchoolAge" => new ChildIsAlmostPrimarySchoolAge($offsetDate, 0),
                 ],
                 'relations' => [],
                 'disqualifiers' => [
-                    new ChildIsSecondarySchoolAge($offsetDate)
+                    "ChildIsPrimarySchoolAge" => new ChildIsPrimarySchoolAge($offsetDate, 0),
                 ],
             ],
-            Family::class => [
+            "App\Family" => [
                 'credits' => [
-                    new FamilyIsPregnant(null, 3)
+                    "FamilyIsPregnant" => new FamilyIsPregnant($offsetDate, 3)
```

Comment:
doesn't a pregnant family need a notice about an upcoming birth?

Reply:
No, we found that could cause issues if the child doesn't get born for some reason...